frame/im-online/src/lib.rs

```diff
@@ -231,7 +231,16 @@ pub trait WeightInfo {
 	fn validate_unsigned_and_then_heartbeat(k: u32, e: u32, ) -> Weight;
 }
 
-pub trait Trait: SendTransactionTypes<Call<Self>> + pallet_session::historical::Trait {
+pub type ValidatorId<T> = <
```

Comment:
Docs would be nice since it's `pub`. Maybe it's sufficient to have this `pub(crate)`?

Reply:
Docs added, but feel free to give it a better description :).